src/main/java/Duke.java

```diff
@@ -1,10 +1,172 @@
+                    break;
+                }
+                if (indexOfBy == 0) {
+                    printEmptyTaskErrorMessage();
+                    break;
+                }
+                taskDescriptionWords = Arrays.copyOf(taskContentWords, indexOfBy);
+                taskDescription = String.join(" ", taskDescriptionWords);
+                String[] taskDeadlineWords = Arrays.copyOfRange(taskContentWords,
+                        indexOfBy + 1, taskContentWords.length);
+                String taskDeadline = String.join(" ", taskDeadlineWords);
+                list[Task.getNumberOfTasks()] = new Deadline(taskDescription, taskDeadline);
+                break;
+            case "event":
+                if (taskContentWords.length == 0) {
+                    printEmptyTaskErrorMessage();
+                    break;
+                }
+                int indexOfAt = -1;
+                for (int i = 0; i < taskContentWords.length; i++) {
+                    if (taskContentWords[i].equals("/at")) {
+                        indexOfAt = i;
+                        break;
+                    }
+                }
+                if (indexOfAt == -1 || indexOfAt == taskContentWords.length - 1) {
+                    printMissingTimeErrorMessage();
+                    break;
+                }
+                if (indexOfAt == 0) {
+                    printEmptyTaskErrorMessage();
+                    break;
+                }
+                taskDescriptionWords = Arrays.copyOf(taskContentWords, indexOfAt);
+                taskDescription = String.join(" ", taskDescriptionWords);
+                String[] taskTimeWords = Arrays.copyOfRange(taskContentWords,
+                        indexOfAt + 1, taskContentWords.length);
+                String taskTime = String.join(" ", taskTimeWords);
+                list[Task.getNumberOfTasks()] = new Event(taskDescription, taskTime);
+                break;
+            case "list":
+                if (taskContentWords.length == 0) {
+                    Task.printList(list);
+                    break;
+                } else {
+                    printConfirmationToListMessage();
+                    String userConfirmation = in.nextLine().toLowerCase(Locale.ROOT);
+                    if (userConfirmation.equals("y")) {
+                        Task.printList(list);
+                    }
+                    break;
+                }
+            case "done":
+                try {
+                    int index = Integer.parseInt(taskContentWords[0]);
+                    list[index - 1].markAsDone();
+                    break;
+                } catch (NumberFormatException | ArrayIndexOutOfBoundsException ex) {
+                    printDoneFormatErrorMessage();
+                    break;
+                } catch (NullPointerException ex) {
+                    printOutOfBoundErrorMessage();
+                    break;
+                }
```

Comment:
`case` should align with `switch` to follow the coding standard provided by cs2113